powa_collector/powa_worker.py

```diff
@@ -207,6 +218,8 @@ def __check_powa(self):
         self.__save_versions()
 
     def __reload(self):
+        """Reload configurationi
+        Disconnect from everything, reconnect, update dependencies, check Powa is still available"""
```

Comment:
we should mention that during reconnect there's no guarantee that the connection will be the same (could have updated user of other stuff)